docker/prodlike/bitcoin/bitcoin.conf

```diff
@@ -0,0 +1,41 @@
+regtest=1 #chain=regtest
+
+[regtest]
+printtoconsole=1
+disablewallet=0
```

Comment:
I think this should be 1 even in regest, I think we want the wallet disabled.

Reply:
Ahhh, this is the same thing we use in devnet. Okay, it's probably fine to use either.

---
Thinking out loud:

Having the wallet enabled is probably necessary in devnet because of the stacks-core minor or for bridge demos. But shouldn't be not necessarily on testnet. Also, our integration tests run bitcoin core with the wallet disabled, and I think it's important during our integration testing. This may explain why I had some trouble running integration tests against devnet.